### ISSUE TITLE
Show number of comments in timeline

### ISSUE THREAD

**Issue by RZHSSNZDH** [cheeaun/phanpy#185]
It would be better to show number of comments of each post without need to click on the post.

**Comment by cheeaun** [comment]
@RZHSSNZDH thanks for suggestion! For now, you can try right-click on the post to see number of comments, like and boost 🙇‍♂️

**Comment by RZHSSNZDH** [comment]
Yes it is possible, but it would be better if there was no need to click on posts to see comments number and do actions like boost.

**Comment by cferdinandi** [comment]
@RZHSSNZDH you can currently long press on touch devices. But I too want this enhancement.

**Comment by RZHSSNZDH** [comment]
@cferdinandi It shows browser's item first. like copy link.
